finality-aleph/src/justification/requester.rs

```diff
@@ -128,7 +128,7 @@ where
                 }
             }
             SchedulerActions::ClearQueue => {
-                debug!(target: "aleph-justification", "Clearing queue");
+                info!(target: "aleph-justification", "Clearing justification request queue");
```

Comment:
This is not a good message to have on level `info`. It doesn't give any useful information about the internal state of the component. What we would like to catch here at least, is that we keep requesting the same block, and fail for a prolonged period of time. In such a case we want to learn the number and hash of the requested block.

Reply:
I guess we could save last num that we requested, save how many times and save a block hash associated with this num. Status report could be called every 10 seconds in component above.

I implemented this in a way, that if there is no pending status report, it should log
```
Justification requester status report: tries - 0;
```
and otherwise, something like that
```
Justification requester status report: tries - 1; requested block number - 5; hash - 0x5d6d…614a;
```

Reply:
Thanks. This looks good now, but could you also make sure the default (i.e. healthy node) is printing nothing, instead of printing some generic message? This could be done by printing only if `tries` crosses some threshold.

Reply:
Is 3 tries a good threshold?

Reply:
Hard to say -- this would be something to figure out. Does it happen to have 3 tries if everything is healthy? What's the typical behavior of this component when things are healthy?

Alternatively, if you don't want to play with that, it's probably fine to always print, but with lower frequency, something like 60 sec should be fine. Also generally I would recommend making separate constants, not share the 10s one between all. It might make sense to increase/reduce them later individually.

Reply:
When things are healthy nothing should be requested more then once.

Typical behavior is such that after three retries without any finalization in mean time, justification queue is cleared. So usually if this is 3 we know, that something might be going on, and we cleared the queue to fix that. 

On topic of frequency, actually 10 seconds might be too frequently. Some of the logs we have here are quite big. Maybe we could just change all of them to 20/30 seconds

Reply:
Yes, you can change others to 20-30 seconds.

And regarding this particular one -- you can make it print only if, say, `tries>=2`. Alternatively, just make it rare, like 60sec, and you can ten print every time.